app/kudos/tasks.py

```diff
@@ -36,7 +36,7 @@ def mint_token_request(self, token_req_id, retry=False):
             obj = TokenRequest.objects.get(pk=token_req_id)
             multiplier = 1
             gas_price = int(float(recommend_min_gas_price_to_confirm_in_time(1)) * multiplier)
-            if gas_price > delay_if_gas_prices_gt_mint:
+            if gas_price > delay_if_gas_prices_gt_mint and self.request.retries < self.max_retries:
```

Comment:
wont this LOC do *more* retries now that you've removed the `self.request.retries < self.max_retries` condition?  now it will infinitely retry, if im reading it right.

Reply:
416e458a43ef1415ac5594afa479b31842c3f0c8

Reply:
https://gitcoincore.slack.com/archives/CAXQ7PT60/p1599159069014200